pilot/pkg/kube/inject/webhook.go

```diff
@@ -687,6 +712,19 @@ func (wh *Webhook) inject(ar *v1beta1.AdmissionReview) *v1beta1.AdmissionRespons
 	return &reviewResponse
 }
 
+func getProxyUID(pod corev1.Pod) (*uint64, error) {
```

Comment:
What about return -1 in cases of errors? That would eliminate the need of dealing with pointers above

Reply:
Yeah, that was my first instinct, too... but IMHO a pointer to represent optionality is much better than using magic values. Not sure what the golang position on this is.